File: python/URI/1036.py
import math
a,b,c=input().split()
a=float(a)
b=float(b)
c=float(c)

delta = (b**2)-(4*a*c)

if a==0 or delta<0:
    print("Impossivel calcular")
else:
    delta=(delta)**0.5
    r1=(-b+delta)/(2*a)
    r2=(-b-delta)/(2*a)
    print("R1 = {:.5f}".format(r1))
    print("R2 = {:.5f}".format(r2))

File: C/URI/1036.c
#include <stdio.h>
#include <math.h>

void main(){
	double A,B,C,delta,R1,R2;
	
	scanf("%lf%lf%lf", &A,&B,&C);
	delta = sqrt(B*B-4*A*C);
	if (A==0 || (B*B-4*A*C)<0)
		printf("Impossivel calcular\n");
	else{
		R1 = ((-B+delta)/(2*A));
		R2 = ((-B-delta)/(2*A));
		printf("R1 = %.5lf\n", R1);
		printf("R2 = %.5lf\n", R2);
	}
	
}